src/appMain/sdl_preloaded_pt.json

```diff
@@ -3914,6 +3918,30 @@
                     "mandatory": false,
                     "since": "5.1"
                 },
+                {
+                    "name": "windowStatus",
+                    "params": [
+                        {
+                            "name": "location",
+                            "key": "OEM_REF_WND_LOC",
+                            "type": "Grid",
+                            "mandatory": true
+                        },
+                        {
+                            "name": "state",
+                            "key": "OEM_REF_WND_STATE",
+                            "type": "WindowState",
+                            "mandatory": true
+                        }
```

Comment:
Referring back to this comment https://github.com/smartdevicelink/sdl_core/pull/3304#discussion_r449029139. The type can only be an enum or a base type. WindowState is a Struct so it should be expanded here.
```suggestion
                        {
                            "name": "state",
                            "params": [
                                {
                                    "name": "approximatePosition",
                                    "key": "OEM_REF_WND_STATE_APRPOS",
                                    "type": "Integer",
                                    "mandatory": true,
                                    "minvalue": 0,
                                    "maxvalue": 100
                                },
                                {
                                    "name": "deviation",
                                    "key": "OEM_REF_WND_STATE_DEV",
                                    "type": "Integer",
                                    "mandatory": true,
                                    "minvalue": 0,
                                    "maxvalue": 100
                                }
                            ],
                            "key": "OEM_REF_WND_STATE",
                            "type": "Struct",
                            "mandatory": true
                        }
```

Reply:
@ShobhitAd Thank you for the review, all comments are processed, please check the latest updates.